prow/bugzilla/fake.go

```diff
@@ -175,7 +193,17 @@ func (c *Fake) CloneBug(bug *Bug) (int, error) {
 	if err != nil {
 		return 0, fmt.Errorf("failed to get parent bug's comments: %v", err)
 	}
-	return c.CreateBug(cloneBugStruct(bug, comments))
+	id, err := c.CreateBug(cloneBugStruct(bug, comments))
+	if err != nil {
+		return id, err
+	}
+	depends := BugUpdate{
+		DependsOn: &IDUpdate{
+			Add: []int{bug.ID},
+		},
+	}
+	err = c.UpdateBug(id, depends)
```

Comment:
Kind of problematic that you have to implement this in in the fake -- you are then testing that the fake is written correctly. not the production client. Let's try to factor this such that it's not the case.